configure.ac

```diff
@@ -66,6 +66,10 @@ AC_ARG_ENABLE([extratests],
 	AS_HELP_STRING([--enable-extratests], [Enable extra tests that may be unreliable on some platforms]))
 AM_CONDITIONAL([ENABLE_EXTRATESTS], [test "x$enable_extratests" = xyes])
 
+AC_ARG_ENABLE([tests],
+	AS_HELP_STRING([--enable-tests], [Enable tests]))
```

Comment:
I think enable_tests should be default to yes.

```
AC_ARG_ENABLE([tests],
        AS_HELP_STRING([--enable-tests], [Enable tests]),
        [],
        [enable_tests=yes])
```

Reply:
it should be good now